roles/rciam-metrics/templates/fastapi/config.py.j2

```diff
@@ -0,0 +1,13 @@
+[database_parameters]
+db = {{ metrics_db }}
+db_admin = {{ metrics_db_admin }}
+db_password = {{ metrics_db_password }}
+database_url = postgresql+psycopg2://{{ metrics_db_admin }}:{{ metrics_db_password }}@{{ metrics_db_hostname }}/{{ metrics_db }}
+
+[oidc_client]
+client_id = {{ oidc_client_id }}
+client_secret = {{ oidc_client_secret }}
+issuer = {{ oidc_client_issuer }}
+openid_connect_url = {{ oidc_client_connect_url }}
```

Comment:
This is redundant. The OP configuration MUST be available under issuer + '/.well-known/blahblah'